draft-ietf-teep-protocol.md

```diff
@@ -1195,8 +1195,8 @@ teep-success = [
 teep-error = [
   type: TEEP-TYPE-teep-error,
   options: {
-     ? token => uint,
-     ? err-msg => text,
+     ? token => bstr .size (8..64),
+     ? err-msg => text (0x00..0x17 bytes),
```

Comment:
I think 23 bytes is too short for a text error message.  The text descriptions of errors at the end of section 4.7 are longer than that.   Those strings wouldn't need to appear literally because we have the error number, but they are illustrative as examples of text error messages that contain enough info to be useful to a developer.  I'd suggest a max of something like 127 bytes.   I'd also suggest the maximum length be mentioned in line 653-654  (text definition of err-msg).

Reply:
I changed the size of `msg` and `err-msg` to maximum 128 bytes rather than 127 bytes, since CBOR do not require null termination at the end of UTF-8 text string.
Also I added the explanation of length in the body.